python/shotgun_fields/shotgun_field_manager.py

```diff
@@ -219,6 +250,29 @@ def create_display_widget(self, sg_entity_type, field_name, entity=None, parent=
 
         return None
 
+    def create_delegate(self, sg_entity_type, field_name, view):
```

Comment:
trying to think ahead about editing - trying to avoid making up use cases but wondering if there is a need for a flag to specify that a delegate should not be editable? Or is this perhaps a property that belongs to the role of the model item?